docs/integrations/hosts-operating-systems/opentelemetry/opentelemetry-collector-insights.md

```diff
@@ -0,0 +1,447 @@
+
+This app supports OpenTelemetry Collector version `0.130.1-sumo-0` and later versions.
+
+We use the OpenTelemetry collector's built-in internal telemetry capabilities to collect metrics and logs about the collector itself. By default, the Collector exposes its own telemetry through internal metrics (via Prometheus interface on port 8888) and logs (emitted to stderr). The collector can also be configured to export its own telemetry data (metrics and logs) to Sumo Logic through OTLP/HTTP endpoints.
+
+:::info
+This app includes [built-in monitors](#opentelemetry-collector-insights-alerts). For details on creating custom monitors, refer to [Create monitors for OpenTelemetry Collector Insights app](#create-monitors-for-opentelemetry-collector-insights-app).
+:::
+
+## Fields creation in Sumo Logic for OpenTelemetry Collector Insights
+
+Following are the [fields](/docs/manage/fields/) which will be created as part of OpenTelemetry Collector Insights app installation, if not already present.
+
+- `sumo.datasource`. Has fixed value of `otel_collector`.
+- `_contentType`. Has fixed value of `OpenTelemetry`.
+- `deployment.environment`. User configured. Enter a name to identify your deployment environment.
+
+## Prerequisites
+
+### For OTLP endpoint configuration
+
+Before configuring the OTEL Collector integration, ensure you have the following prerequisites in place:
+
+1. **Sumo Logic OTLP Source**: You need to create an OTLP source in your Sumo Logic hosted collector. The OTLP source will provide the endpoint URL that the OTEL Collector will use to send telemetry data.
+
+  **Documentation**: [Creating a Sumo Logic OTLP Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/otlp/)
+
+### For metrics collection
+
+The OpenTelemetry Collector must be configured to export its own metrics using the built-in telemetry capabilities. This requires:
+- OpenTelemetry Collector version 0.130.1-sumo-0 or later
+- Collector configured with telemetry metrics enabled at `detailed` level
+- Access to OTLP endpoint for metrics export
+- Internal metrics exposed on port 8888 (default)
+
+### For logs collection
+
+The OpenTelemetry Collector must be configured to export its own logs using the built-in telemetry capabilities. This requires:
+- Collector configured with telemetry logs enabled at `debug` level
+- JSON encoding for structured log output
+- Access to OTLP endpoint for logs export
+
+### System Requirements
+
+- OTEL Collector v0.130.1-sumo-0 or later
+- Sufficient system resources (CPU, memory) for data processing
+- Proper permissions for the collector service to access configured resources
+
+import LogsCollectionPrereqisites from '../../../reuse/apps/logs-collection-prereqisites.md';
```

Comment:
Does log collection prerequisite hold true here ? Since most the content inside it assumes that the logs are collected via filelog receiver and instruction are around its permission etc. Can you check and change this accordingly